src/main/java/controller/PlayerController.java

```diff
@@ -0,0 +1,206 @@
+package controller;
+
+import entity.Player;
+import entity.Team;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.text.Normalizer;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Pattern;
+
+@RestController
+@RequestMapping("/player-service")
```

Comment:
```suggestion
@RequestMapping("/player")
```

Yo lo llamaría solo player ya que este controller se encargará solo de acciones relacionadas a los Players